Sources/TSCLibc/libc.swift

```diff
@@ -29,10 +29,45 @@ public func realpath(
 }
 
 // char *mkdtemp(char *template);
+// NOTE(compnerd) this is unsafe!  This assumes that the template is *ASCII*.
 public func mkdtemp(
     _ template: UnsafeMutablePointer<CChar>?
 ) -> UnsafeMutablePointer<CChar>? {
-  fatalError("mkdtemp is unimplemented")
```

Comment:
was this just not used before?

Reply:
It "was" in the sense that s-p-m never got to the point of building stuff previously.  I've had this change laying around for ages.  Just draining changes that will allow s-p-m to get further :)

Reply:
ah gotcha